library.rb

```diff
@@ -0,0 +1,34 @@
+require 'yaml'
+require './utils.rb'
+class Library
+
+  def initialize(datafile)
+    @utils = Utils.new(datafile)
+    @lib = @utils.load_file  
+  end
+
+  def most_popular_book
+    puts @lib[:orders].group_by {|orders| orders[:book]}.sort_by {|key, value| value.length}.last.first
+  end
+
+  def often_takes_book
+    puts @lib[:orders].group_by {|orders| orders[:reader]}.sort_by {|key, value| value.length}.last.first
+  end
+
+  def three_popular_books
+    b =  @lib[:orders].group_by {|orders| orders[:book]}.sort_by {|key, value| value.length}.last(3)
```

Comment:
1. Rename `b` to `books` (always use understandable name of variables)
2. Use `.max_by(3) { |_book, orders| orders.size }`